src/job.rs

```diff
@@ -19,14 +105,59 @@ impl Job {
                     .collect::<Vec<&str>>()
                     .as_slice(),
             )
+            .current_dir(&work_dir)
+            // TODO: this is going to have to retain some environment variables
+            // for software to work correctly. For example, we'll probably need
+            // to provide a fake HOME the way Nix does.
```

Comment:
Good point! Also `PATH` perhaps? And I think current working directory is an environment variable on some (all?) OSes we'll be targeting.

Reply:
resolving because talked live about this